docs/configuration.md

```diff
@@ -472,15 +472,17 @@ Apart from these, the following properties are also available, and may be useful
   <td>20</td>
   <td>
     Default number of rows in eager evaluation output HTML table generated by <code>_repr_html_</code> or plain text,
-    this only take effect when <code>spark.sql.repl.eagerEval.enabled</code> is set to true.
+    this only takes effect when <code>spark.sql.repl.eagerEval.enabled</code> is set to true. The valid range of this
```

Comment:
For SQLConf, we do not need to hard code the conf description here.

Reply:
Got it, thanks, done in afada2b.